echopype/tests/echodata/test_echodata_combine.py

```diff
@@ -112,38 +110,52 @@ def test_combine_echodata(raw_datasets):
     for group_name, value in combined.group_map.items():
         if group_name in ("top", "sonar", "provenance"):
             continue
-        try:
-            combined_group: xr.Dataset = combined[value['ep_group']]
-            eds_groups = [
-                ed[value['ep_group']]
-                for ed in eds
-                if ed[value['ep_group']] is not None
-            ]
-
-            def union_attrs(datasets: List[xr.Dataset]) -> Dict[str, Any]:
-                """
-                Merges attrs from a list of datasets.
-                Prioritizes keys from later datasets.
-                """
-
-                total_attrs = {}
-                for ds in datasets:
-                    total_attrs.update(ds.attrs)
-                return total_attrs
-
-            test_ds = xr.combine_nested(
-                eds_groups,
-                [concat_dims.get(group_name, concat_dims["default"])],
-                data_vars=concat_data_vars.get(
-                    group_name, concat_data_vars["default"]
-                ),
-                coords="minimal",
-                combine_attrs="drop",
-            )
-            test_ds.attrs.update(union_attrs(eds_groups))
-            test_ds = test_ds.drop_dims(
+        combined_group: xr.Dataset = combined[value['ep_group']]
+        eds_groups = [
+            ed[value['ep_group']]
+            for ed in eds
+            if ed[value['ep_group']] is not None
+        ]
+
+        def union_attrs(datasets: List[xr.Dataset]) -> Dict[str, Any]:
+            """
+            Merges attrs from a list of datasets.
+            Prioritizes keys from later datasets.
+            """
+
+            total_attrs = {}
+            for ds in datasets:
+                total_attrs.update(ds.attrs)
+            return total_attrs
+
+        test_ds = xr.combine_nested(
+            eds_groups,
+            [concat_dims.get(group_name, concat_dims["default"])],
+            data_vars=concat_data_vars.get(
+                group_name, concat_data_vars["default"]
+            ),
+            coords="minimal",
+            combine_attrs="drop",
+        )
+        test_ds.attrs.update(union_attrs(eds_groups))
+        test_ds = test_ds.drop_dims(
+            [
+                # xarray inserts this dimension when concatenating along multiple dimensions
```

Comment:
```suggestion
                # xarray inserts "concat_dim" when concatenating along multiple dimensions
```
Just to make thing painfully clear